core/templates/dev/head/filters.js

```diff
@@ -342,6 +342,26 @@ oppia.filter('summarizeNonnegativeNumber', [function() {
   };
 }]);
 
+// Note that this filter removes additional new lines or <p><br></p> tags
+// at the end of the string.
+oppia.filter('removeExtraLines', [function() {
+  return function(string) {
+    if (!angular.isString(string)) {
+      return string;
+    }
+    while (1) {
+      var lastIndex = string.lastIndexOf('>');
+      if (string.substring(lastIndex - 10, lastIndex + 1) === '<p><br></p>') {
```

Comment:
Just to make this more readable, I suggest assigning `var BLANK_LINE_TAGS = '<p><br></p>` at the start of this function. Then instead of `10` you can use `BLANK_LINE_TAGS.length -1`.